modules/lang-mustache/src/main/java/org/elasticsearch/script/mustache/MustacheScriptEngine.java

```diff
@@ -79,6 +80,11 @@
 
     }
 
+    @Override
+    public List<ScriptContext<?>> getSupportedContexts() {
+        return List.of(TemplateScript.CONTEXT);
```

Comment:
Note that if we backport this to a version requiring Java 8, List.of wasn't introduced yet.

Reply:
I intend to address that in the backport.  No reason to avoid using it here.